css/custom.css

```diff
@@ -3805,7 +3805,7 @@ img.emoji {
 
 }
 
-@media only screen and (max-width: 48rem) {
+@media only screen and (max-width: 24rem) {
```

Comment:
so after looking into this, what we're doing here is reducing the point where our screens width triggers the "roadmaps mobile view" (which isn't the best) to display by 50% and then we view the original/landscape roadmap, so any visual edits here will carry over to desktop view / vice-versa.

maybe we can move away from max-width and use `@media (orientation: landscape)`
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation

Reply:
> so after looking into this, what we're doing here is reducing the point where our screens width triggers the "roadmaps ~~mobile~~ inline view" (which isn't the best) to display by 50%* and then we view the original/landscape roadmap,

correct
> so any visual edits here will carry over to desktop view / vice-versa.

not quite

The edits are purely based on the size of the browser window (responsive). On desktop, you can trigger the different levels by resizing the browser window. - these changes don't effect "desktop" or landscape views. It delays the switch of the portrait "tabbed view" to "inline" til a much smaller screen width. 
(the inline view is trash.)

> *im still wondering how the roadmap mobile view styles are applied 

based on the size of the browser window/viewport

> maybe we can move away from max-width and use `@media (orientation: landscape)`
> https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation

This doesn't work iiuc, because activating the keyboard will make portrait appear to be landscape